content/04.results.md

```diff
@@ -70,15 +71,15 @@ Preprints from certain topic areas that spanned approaches from informatics-rela
 **B.** Potential preprint-publication pairs that are unannotated but within the 50th percentile of all preprint-publication pairs in the document embedding space are likely to represent true preprint-publication pairs.
 We depict the fraction of true positives over the total number of pairs in each bin.
 Accuracy is derived from the curation of a randomized list of 200 potential pairs (50 per quantile) performed in duplicate with a third rater used in the case of disagreement.
-**C.** Most preprints are eventually published.
+**C.** Most preprints are eventually published even when adjusting for preprints posted in 2019 and on.
 We show the publication rate of preprints since bioRxiv first started.
 The x-axis represents months since bioRxiv started, and the y-axis represents the proportion of preprints published given the month they were posted.
 The light blue line represents the publication rate previously estimated by Abdill et al. [@doi:10.7554/eLife.45133].
 The dark blue line represents the updated publication rate using only CrossRef-derived annotations, while the dark green line includes annotations derived from our embedding space approach.
```

Comment:
I think that you may not be comparing apples to apples here because Abdill et al is an earlier snapshot. How are you making sure that your comparison with Abdill is appropriate?

Reply:
>How did you deal with Abdill et al?
>
>Is this the publication rate that they estimated or one that you derived from their estimates?
>
>I think that you may not be comparing apples to apples here because Abdill et al is an earlier snapshot. How are you making sure that your comparison with Abdill is appropriate?

I used their estimated rate initially. Didn't catch that Abdill et al. didn't adjust for recency in their work (used every preprint in their calculation rather than filtering out preprints posted in 2018). In light of this discovery a fair comparison is to adjust for recency like we did for our current snapshot. Otherwise I'd have to change the response to reviewers to state why I didn't adjust for recency (e.g. keep the comparison fair between us and Abdill et al.).